operations/security_and_authentication/Audit_Logs.md

```diff
@@ -0,0 +1,146 @@
+      It is not required to enable both.
+
+      * During CSM install, from the PIT node
+
+        **NOTE:** This step needs to happen at the same time that `csi config init` is normally run during the install.
+
+        To update the audit log settings during the installation, use one of the following options:
+
+        * **Option 1**
+
+          During the installation, audit logging is enabled or disabled by modifying the CSI settings.
+          To enable or disable audit logging, use the following flags with the `csi config init` command. For more information on using flags, see `csi config init -h`.
+
+          * `ncn-mgmt-node-auditing-enabled`
+
+            Set to `true` to enable host logging or to `false` to disable host logging.
+
+            ```console
+            pit# csi config init --ncn-mgmt-node-auditing-enabled=true [other config init options]
+            ```
+
+          * `k8s-api-auditing-enabled`
+
+            Set to `true` to enable Kubernetes API logging or to `false` to disable Kubernetes API logging.
+
+            ```console
+            pit# csi config init --k8s-api-auditing-enabled=true [other config init options]
+            ```
+
+        * **Option 2**
+
+          Adjust the audit log settings by editing the `system_config.yaml` file.
+
+          View the current settings with the following command:
+
+          ```console
+          pit# cd /var/www/ephemeral/prep
+          pit# grep audit system_config.yaml
+          ```
+
+          Example output:
+
+          ```text
+          k8s-api-auditing-enabled: false
+          ncn-mgmt-node-auditing-enabled: false
+          ```
+
+      * After CSM install
+
+        * **Option 1**
+
+          Enable audit logging through the following `csi` command.
+
+          * `ncn-mgmt-node-auditing-enabled`
+
+            ```console
+            ncn-m001# TOKEN=$(curl -k -s -S -d grant_type=client_credentials -d client_id=admin-client \
+                           -d client_secret=`kubectl get secrets admin-client-auth -o jsonpath='{.data.client-secret}' | base64 -d` \
+                           https://api-gw-service-nmn.local/keycloak/realms/shasta/protocol/openid-connect/token | jq -r '.access_token') \
+                           csi handoff bss-update-param --limit <mgmt-node-xname> --set ncn-mgmt-node-auditing-enabled=true
+            ```
+
+          * `k8s-api-auditing-enabled`
+
+            ```console
+            ncn-m001# TOKEN=$(curl -k -s -S -d grant_type=client_credentials -d client_id=admin-client \
+                           -d client_secret=`kubectl get secrets admin-client-auth -o jsonpath='{.data.client-secret}' | base64 -d` \
+                           https://api-gw-service-nmn.local/keycloak/realms/shasta/protocol/openid-connect/token | jq -r '.access_token') \
+                           csi handoff bss-update-param --limit <mgmt-node-xname> --set k8s-api-auditing-enabled=true
+            ```
```

Comment:
In the case of the PIT node scenario, csi will definitely be there. And now that we are providing non-CSI options for post-install, I think the install CSI step should be moved. Something like this:

````suggestion
   1. Select one of the following options to enable audit logging based on the installation status of the system:

      For each of the following options, only enable the desired level of audit logging.
      It is not required to enable both.

      * During CSM install, from the PIT node

        **NOTE:** This step needs to happen at the same time that `csi config init` is normally run during the install.

        To update the audit log settings during the installation, use one of the following options:

        * **Option 1**

          During the installation, audit logging is enabled or disabled by modifying the CSI settings.
          To enable or disable audit logging, use the following flags with the `csi config init` command. For more information on using flags, see `csi config init -h`.

          * `ncn-mgmt-node-auditing-enabled`

            Set to `true` to enable host logging or to `false` to disable host logging.

            ```console
            pit# csi config init --ncn-mgmt-node-auditing-enabled=true [other config init options]
            ```

          * `k8s-api-auditing-enabled`

            Set to `true` to enable Kubernetes API logging or to `false` to disable Kubernetes API logging.

            ```console
            pit# csi config init --k8s-api-auditing-enabled=true [other config init options]
            ```

        * **Option 2**

          Adjust the audit log settings by editing the `system_config.yaml` file.

          View the current settings with the following command:

          ```console
          pit# cd /var/www/ephemeral/prep
          pit# grep audit system_config.yaml
          ```

          Example output:

          ```text
          k8s-api-auditing-enabled: false
          ncn-mgmt-node-auditing-enabled: false
          ```

      * After CSM install

        * **Option 1**

          Enable audit logging using the `csi` tool.

          1. (Optional) If the `csi` command is not installed, locate the `cray-site-init` RPM on `ncn-m001` and install it.

             ```console
             ncn-m001# find /mnt/pitdata -name cray-site-init*
             ncn-m001# rpm -Uvh --force <rpm file path>
             ```

          1. Enable audit logging.

             * `ncn-mgmt-node-auditing-enabled`

               ```console
               ncn-m001# TOKEN=$(curl -k -s -S -d grant_type=client_credentials -d client_id=admin-client \
                                   -d client_secret=`kubectl get secrets admin-client-auth -o jsonpath='{.data.client-secret}' | base64 -d` \
                                   https://api-gw-service-nmn.local/keycloak/realms/shasta/protocol/openid-connect/token | jq -r '.access_token') \
                                   csi handoff bss-update-param --limit <mgmt-node-xname> --set ncn-mgmt-node-auditing-enabled=true
               ```

             * `k8s-api-auditing-enabled`

               ```console
               ncn-m001# TOKEN=$(curl -k -s -S -d grant_type=client_credentials -d client_id=admin-client \
                                   -d client_secret=`kubectl get secrets admin-client-auth -o jsonpath='{.data.client-secret}' | base64 -d` \
                                   https://api-gw-service-nmn.local/keycloak/realms/shasta/protocol/openid-connect/token | jq -r '.access_token') \
                                   csi handoff bss-update-param --limit <mgmt-node-xname> --set k8s-api-auditing-enabled=true
               ```
````